src/test/java/com/firebolt/jdbc/type/array/SqlArrayUtilTest.java

```diff
@@ -118,6 +119,41 @@ void shouldTransformNullArrayToNull(String type) throws SQLException {
 		assertNull(SqlArrayUtil.transformToSqlArray("NULL", ColumnType.of(type)));
 	}
 
+	@ParameterizedTest
+	@ValueSource(booleans = {true, false})
+	void nullByteArrayToString(boolean separateEachByte) {
+		assertNull(SqlArrayUtil.byteArrayToHexString(null, separateEachByte));
+	}
+
+	@ParameterizedTest
+	@CsvSource({
+			"ABC,false,\\x414243",
+			"abc,true,\\x61\\x62\\x63"
```

Comment:
Let's add some special cases like `\`, newline character and space.

Reply:
Done and even fixed bug. 10x.

Reply:
Let's add slash character for completeness. e.g. `AB\C' just to make sure that slash is not lost in any way.

Reply:
Added both slash `/` and back slash `\`